resources/jscomposition/cases/casesDetail/config/columns.js

```diff
@@ -93,23 +99,41 @@ const requestNameColumn = () => ({
 });
 
 const currentTaskColumn = () => ({
-  field: "current_task",
-  header: "Current Task",
+  field: "active_tasks",
+  header: "Task",
   resizable: true,
   width: 200,
+  formatter:(row, column, columns)=>{
+    return row.active_tasks.length? row.active_tasks[0].element_name : "";
+  },
+  cellRenderer: () => ({
+    component: TruncatedOptionsCell,
+    params: {
+      click: (option, row, column, columns) => {
+        window.document.location = `/tasks/${option.id}/edit`;
+      },
+      formatterOptions:(option, row, column, columns)=>{
+        return option.element_name;
+      }
+    },
+  }),
 });
 
```

Comment:
The column participants is missing
![image](https://github.com/user-attachments/assets/1664711a-887a-4617-8e7d-f16c78d3a73b)